docs/docs/cypher_support.md

```diff
@@ -81,100 +81,18 @@ We do not support any of these properties at the type level, meaning nodes and r
 + UNION ALL
 
 ## Functions
-### Scalar functions
-+ id
-+ labels
-+ timestamp
-+ type
-+ coalesce
-+ startNode
-+ endNode
-+ [list comprehensions](/commands/graph.query/#list-comprehensions)
-+ toBoolean
-+ toBooleanOrNull
 
-  **Unsupported:**
-
-- Temporal arithmetic functions
-- Functions returning maps (properties)
-
-### Aggregating functions
-+ avg
-+ collect
-+ count
-+ max
-+ min
-+ percentileCont
-+ percentileDist
-+ stDev
-+ stDevP
-+ sum
-
-### List functions
-+ head
-+ range
-+ reverse
-+ size
-+ tail
-
-### Math functions - numeric
-+ abs
-+ ceil
-+ floor
-+ sign
-+ round
-+ rand
-+ toInteger
-+ toIntegerOrNull
-+ toFloat
-+ toFloatOrNull
-
-### String functions
-+ left
-+ right
-+ trim
-+ lTrim
-+ rTrim
-+ reverse
-+ substring
-+ toLower
-+ toUpper
-+ toString
-+ toStringOrNull
-+ split
+The currently-supported functions can be found in [the Functions documentation](/commands/graph.query/#functions).
 
   **Unsupported:**
 
-- replace
-
-### Predicate functions
-+ exists
-+ any
-+ all
-+ single
-+ none
-
-### Expression functions
-+ case...when
-
-### Geospatial functions
-+ distance
-+ point
-
-### Unsupported function classes
-
-- Logarithmic math functions
-- Trigonometric math functions
+- Temporal arithmetic functions
 - User-defined functions
 
 ## Operators
 
 ### Mathematical operators
-+ Multiplication, addition, subtraction, division, modulo
-
-  **Unsupported:**
-
-- Exponentiation
++ Multiplication, addition, subtraction, division, modulo, power
```

Comment:
it seems like `Multiplication, addition, subtraction, division, modulo, power` is marked as `Unsupported`
see line 146/87

Reply:
No... They were listed as supported.
Anyway, I changed it to point to GRAPH.QUERY for mathematical operators as well.